JS.md

```diff
@@ -361,6 +365,7 @@ each [external](https://github.com/WebAssembly/spec/blob/master/interpreter/spec
 1. If `e` is a [global](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/instance.ml#L15) `v`:
   1. [Assert](https://tc39.github.io/ecma262/#assert): the global is immutable
      by MVP validation constraint.
+  1. If `v` is an `i64`, throw a `WebAssembly.LinkError`.
```

Comment:
Oh, this actually means that you cannot export an `i64` on the Web, not even to another Wasm module. Rather unfortunate. Should we do something about that?